app/uk/gov/hmrc/helplinefrontend/controllers/CallHelpdeskController.scala

```diff
@@ -55,7 +59,35 @@ class CallHelpdeskController @Inject()(implicit
         logger.warn(s"[VER-517] calling without a valid help key($helpKey): request.headers => ${request.headers}")
         Future.successful(Ok("the page being built in VER-592"))
     }
+  }
+
+  def callOptionsNoAnswersPage(): Action[AnyContent] = Action.async { implicit request =>
+        logger.debug(s"[VER-539] Showing options for ${ appConfig.callOptionsList.mkString(", ")}")
+
+    Future.successful(Ok(callOptionsNoAnswers(callOptionform)))
+  }
+
+
+  def oneOfConstraint[T](options: Seq[T], error: String = "error.invalid"): Constraint[T] = Constraint { v ⇒
```

Comment:
you might need move this method in to a service class? you know create a package name it as services